src/pages/Pool/RemoveLiquidity.tsx

```diff
@@ -128,6 +123,8 @@ export default function RemoveLiquidity({ token0, token1 }: { token0: string; to
   const pair = usePair(inputToken, outputToken)
   const pairContract: Contract = usePairContract(pair?.liquidityToken.address)
 
+  console.log(pair?.liquidityToken?.address)
```

Comment:
Just curious if you meant to leave this logging statement in here?

Reply:
nope, fixed